packages/mint-components/src/utilities.ts

```diff
@@ -30,3 +30,11 @@ export function isMobile(breakPoint: number): boolean {
 export function isValidColor(teststr: string) {
   return CSS.supports(`(color: ${teststr})`);
 }
+
+export function sanitizeUrlPath(path: string): string {
+  const url = new URL(path, window.location.href);
+  const cleanUrl = new URL(window.location.href);
+  cleanUrl.pathname = url.pathname;
+
+  return cleanUrl.pathname;
+}
```

Comment:
Allow `search` to be included as well instead of only path.

```suggestion
export function sanitizeUrlPath(path: string): URL {
  const url = new URL(path, window.location.href);
  const cleanUrl = new URL(window.location.href);
  cleanUrl.pathname = url.pathname;
  cleanUrl.search = url.search;
  return cleanUrl;
}
```